build/toolchain/host/BUILD.gn

```diff
@@ -33,14 +34,12 @@ gcc_toolchain("${host_os}_${host_cpu}_clang") {
   }
 }
 
-if (defined(chip_fake_plaform)) {
-  gcc_toolchain("fake_${host_cpu}_gcc") {
-    toolchain_args = {
-      current_os = host_os
-      current_os = host_os
-      current_cpu = host_cpu
-      is_clang = false
-      chip_fake_platform = true
-    }
+gcc_toolchain("fake_${host_cpu}_gcc") {
```

Comment:
Fix it by moving this to a separate build file.

Reply:
Done